CHAP/runner.py

```diff
@@ -12,6 +12,12 @@
 # local modules
 from CHAP.pipeline import Pipeline
 
+# load user processor
+try:
+    import users
+except ImportError:
+    pass
+
```

Comment:
Why try importing the CHAPaaS users module here, since the specific UserProcessor configured in a Pipeline will be imported just before creating the corresponding object?